finetuner/helper.py

```diff
@@ -52,15 +53,30 @@ def get_framework(dnn_model: AnyDNN) -> str:
     :return: `keras`, `torch`, `paddle` or ValueError
 
     """
-    if 'keras' in dnn_model.__module__:
-        return 'keras'
-    elif 'torch' in dnn_model.__module__:  # note: cover torch and torchvision
-        return 'torch'
-    elif 'paddle' in dnn_model.__module__:
-        return 'paddle'
-    else:
+
+    framework = None
+
+    if importlib.util.find_spec('torch'):
+        import torch
+
+        if isinstance(dnn_model, torch.nn.Module):
+            framework = 'torch'
+    if framework is None and importlib.util.find_spec('torch'):
+        import paddle
+
+        if isinstance(dnn_model, paddle.nn.Layer):
+            framework = 'paddle'
+    if framework is None and importlib.util.find_spec('torch'):
```

Comment:
this is tf.

Reply:
Ah thanks.... as for how tests pass - we have all frameworks installed. No way to test this properly unless we have an environment with only one framework- but that's too much effort for only 1 test function

Reply:
Fixed now @bwanglzu